### PR TITLE
Was not compatible with jquery when getting values from dataset

### PR DESCRIPTION
**Pull request by alphons**
Now it behaves like jquery.

### CODE REVIEW

File: src/$.js [hunk 1]
```diff
@@ -94,7 +94,7 @@
 
                     // if the selector is an ID
                     // select the matching element and add it to the elements array
-                    if (selector.match(/^\#[^\s]+$/)) elements.push(parent.getElementById(selector.substr(1)));
```
Comment (stefangabos) [inline src/$.js]: the selector is supposed to be an ID. in this case, `getElementById` is is faster than `querySelector`

Comment (alphons) [reply src/$.js]: Yes, but only document has getElementById , other dom elements have not!

Comment (alphons) [reply src/$.js]: change parent to document, is also okay, but when having 2 same ID;s in the document .... the parent is more usefull, and compatible with jquery

**Comment by stefangabos** [comment]
Fixed by 3232344d3fdb2164016d86cf1752889ee3966172

**Pull request closed without merging**